hw3/index.html

```diff
@@ -0,0 +1,193 @@
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Home</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Portfolio</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">About</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Services</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Projects</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Contact</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+	<div class="banner">
+		<div class="banner__images">
+			<img src="image/banner-1.png" alt="" class="banner__img banner__img-1">
+			<img src="image/banner-2.png" alt="" class="banner__img banner__img-2">
+			<img src="image/banner-3.png" alt="" class="banner__img banner__img-3">
+		</div>
+		<div class="banner__info">
+			<div class="banner-container">
+				<div class="banner__title">Lorest Nesto Gravido Forum</div>
+				<div class="banner__text">Duis porta ultrices elit sed consequat. Suspendisse nunc orci, molestie sed mollis nec, facilisis at 	ligula. Donec ultrices varius lacinia. Pellentesque id tortor non justo consequat pulvinar. </div>
+				<button type="submit" class="banner__button banner__button_button">See More</button>
+			</div>
+		</div>
+	</div>
+	<div class="subscription">
+		<div class="subscription__text">Duis porta ultrices elit sed consequat. Suspendisse nunc orci molestie sed.</div>
+		
+		<form class="subscription__email">
+			<input class="subscription__email" type="email" name="email" placeholder="Email">
+			<input class="subscription__submit subscription__submit_blue" type="submit" name="submit" value="Submit">
+		</form>
+	</div>
+	<div class="preview">
+		<div class="preview__center">
+			<section class="preview__item preview__item-top">
+				<div class="preview__container">
+					<img src="image/icon-1.png" class="preview__logo">
+					<div class="preview__title">
+						ultricies lacinia pulvinar
+					</div>
+					<div class="preview__text">
+						Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.
+					</div>
+				</div>
+				<button class="preview__button">See more</button>
+			</section>
+			<section class="preview__item preview__item-top">
+				<div class="preview__container">
+					<img src="image/icon-2.png" class="preview__logo">
+					<div class="preview__title">
+						magnust minima sed quia
+					</div>
+					<div class="preview__text">
+						Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea asekin morir commodi consequatur?
+					</div>
+				</div>
+				<button class="preview__button">See more</button>
+			</section>
+			<section class="preview__item preview__item-top">
```

Comment:
don't use top modifier, customer could deside to change the order of this blocks and then you classname will seem silly. Use like --smal-img, or --with-btn